pootle/apps/virtualfolder/migrations/0013_set_projects_languages.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+from pootle.core.url_helpers import split_pootle_path
+
+
+def parse_filter_rule(dir_path, filter_rule, lang_from_filters, proj_from_filters):
+    filter_rule = "/%s/" % filter_rule.strip("/")
+    rule_lang = rule_proj = None
+    new_rule = filter_rule
+    if lang_from_filters:
+        rule_lang, rule_proj = split_pootle_path(filter_rule)[:2]
+        new_rule = filter_rule[len(rule_proj) + len(rule_lang) + 2:]
+    elif proj_from_filters:
+        rule_proj = split_pootle_path("/X/%s/" % filter_rule)[1]
```

Comment:
If the original rule is for example `firefox/dir/file.po`, then  `filter_rule` would be `/firefox/dir/file.po/`, and the interpolation in here will yield `/X//firefox/dir/file.po/`, therefore `rule_proj` will be `""` (empty string).